client/components/domains/domain-suggestion-flag/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+var React = require( 'react' );
+
+/**
+ * Internal dependencies
+ */
+var Flag = require( 'components/flag' );
+
+const DomainSuggestionFlag = React.createClass( {
+	propTypes: {
+		domain: React.PropTypes.string
+	},
+
+	render() {
+		var newTLDs = ['.live'];
+		var i;
+
+		if ( this.props.domain ) {
+			for ( i = 0; i < newTLDs.length; i ++ ) {
+				if ( this.props.domain.endsWith( newTLDs[i] ) ) {
+					return (
+						<Flag
+							type="is-success">
+							{ this.translate( 'New' ) }
+						</Flag>
+					);
+				}
+			}
+		}
+
+		return null;
+	}
+} );
+
+module.exports = DomainSuggestionFlag;
```

Comment:
You might as well use ES6 here too (since you use `const`, etc. above) and write it as `export default DomainSuggestionFlag;` :)